### PR TITLE
Draft 'How can my Business Accept Dogecoin' Article

### PR DESCRIPTION
**Pull request by RichieJohn**
Not sure whether it's worthwhile to end this article by flagging the Foundation's plans to develop a L2 POS app? For example:

The Dogecoin Foundation have also flagged [plans] (https://foundation.dogecoin.com/trailmap/prologue/) to develop a Point of Sale (POS) layer 2 application for merchants. Etc, etc.

### CODE REVIEW

**Comment by MishaBoar** [comment]
> Not sure whether it's worthwhile to end this article by flagging the Foundation's plans to develop a L2 POS app? For example:
> 
> The Dogecoin Foundation have also flagged [plans] (https://foundation.dogecoin.com/trailmap/prologue/) to develop a Point of Sale (POS) layer 2 application for merchants. Etc, etc.

We need @tjstebbing's opinion on this. 

Also, do you think we could/should link to the Coinbase and Bitpay Payment pages on their websites? I normally share those links when users ask this question and want a centralized solution like Coinbase or Bitpay.

**Comment by RichieJohn** [comment]
> We need @tjstebbing's opinion on this.

OK, great!

> Also, do you think we could/should link to the Coinbase and Bitpay Payment pages on their websites? I normally share those links when users ask this question and want a centralized solution like Coinbase or Bitpay.

I have no strong opinion on this, just mindful of potentially 'overly endorsing' if direct links are provided. But, again, no strong opinion.

**Review by xanimo — CHANGES_REQUESTED** [review]
would suggest adding a statement in italics or footnote that says something like 'dogecoin fdn is not affiliated with bitpay and coinbase, etc. in any way.'

File: content/dogepedia/articles/how-can-my-business-accept-dogecoin.md [hunk 1]
```diff
@@ -11,4 +11,8 @@ At the most foundational level, for your business to accept Dogecoin in exchange
 
 If you have a bricks-and-mortar business where payments are made in person, it’s common for merchants to print a QR code that is scanned by customers’ smartphones to streamline the transaction process.  
 
-If you have an ecommerce store, companies like Coinbase Commerce and BitPay make accepting Dogecoin a breeze, and include integrations with popular platforms like Shopify, WooCommerce, and Opencard!
\ No newline at end of file
+If you have an ecommerce store, companies like Coinbase Commerce and BitPay make accepting Dogecoin a breeze, and include integrations with popular platforms like Shopify, WooCommerce, and Opencard!
+
+***
+
+*Dogecoin and the Dogecoin Foundation is not affiliated with the companies mentioned in this article.*
```
Comment (xanimo) [inline content/dogepedia/articles/how-can-my-business-accept-dogecoin.md:18]: i think the disclaimer should be 'are not affiliated' instead of is?

Comment (RichieJohn) [reply content/dogepedia/articles/how-can-my-business-accept-dogecoin.md]: Agreed. Changed. :)

**Review by xanimo — APPROVED** [review]

**Pull request merged**